.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+## 📝 Description
+
+Please include a summary of the changes and the related issue number(s). Provide context and motivation for the change, and describe what problem it solves.
+
+- **Issue:** [Link](#)
```

Comment:
Is `Link` for the Notion ticket? If so, the Notion / GitHub integration will add it automatically, no?

Reply:
Ah I see, I didn't understand because I never use issues, even though we have a few open. Up to you :)